### DISCUSSION TITLE
Not able to use AutoCompletion in Vue 2

### DISCUSSION THREAD

**Discussion by praveshishere** [zed-industries/zed#7627] [Feature Requests]
Hi, I am trying to use Zed for a Vue 2 project, though it doesn't seem to work.
I tried it with Vue 3 (on Nuxt), which worked great.

I was curious to know if Zed doesn't support vue2 or if I need to configure it somehow to make it work.

Thanks for any suggestions

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
